### ISSUE TITLE
Authentication Profile patch

### ISSUE THREAD

**Issue by cmgpbint** [GoogleCloudPlatform/application-integration-management-toolkit#148]
When we are trying to use patch for authentication profiles it throws an error, the same payload works well via REST API call

```
<p>The requested URL <code>/v1/projects/<PROJECT>/locations/europe-west3/connections?updateMask=%2A</code> was not found on this server. 

 <ins>That’s all we know.</ins>
```

**Comment by ssvaidyanathan** [comment]
@cmgpbint - can you please share the command you are executing

**Comment by cmgpbint** [comment]
`$(apigeeCONFIGS_INTEGRATIONS_CLI_GET) authconfigs update \
                                        -t $(apigeeCONFIGS_INTEGRATIONS_CLIToken_GET) \
                                        -f $AUTH_PROFILE_PAYLOAD \
                                        -r $REGION \
                                        --update-mask "*" \
                                        -p $PROJECT`
                                        
                                        
We are using a bash framework to encapsulate, but the arguments are being set correctly

**Comment by ssvaidyanathan** [comment]
which version are you using?

```sh
integrationcli --version
```

**Comment by cmgpbint** [comment]
We use a image, I had check and we are on 0.69

`RUN export INTEGRATIONCLI_VERSION=v0.69 &&\
    curl -L https://raw.githubusercontent.com/GoogleCloudPlatform/application-integration-management-toolkit/main/downloadLatest.sh | sh -  `
    
    We were forced to set a version because of the LTS support, we had issues when we used downloadLatest.sh script

**Comment by ssvaidyanathan** [comment]
Thanks for confirming. There was a fix for Auth config update in v0.70 (https://github.com/GoogleCloudPlatform/application-integration-management-toolkit/releases/tag/v0.70)

Please update your version and retry. Let us know if the issue still exist

**Comment by cmgpbint** [comment]
Thanks for the feedback, I'll give you an update with LTS 0.70

**Comment by ssvaidyanathan** [comment]
@cmgpbint - were you able to test?

**Comment by cesar-m-pinto-alb** [comment]
Yes, after upgrade tthe tools to latest version ( _integrationcli version 0.70 date: 2023-10-04T01:12:26Z [commit: 7968f90]_ )

We get the same error, 

```
<p>The requested URL <code>/v1/projects/<PROJECT>/locations/europe-west3/connections?updateMask=%2A</code> was not found on this server. 

 <ins>That’s all we know.</ins>
```

worth menction when using via REST we can do it. The URL is sighly different, we are using

`https://%s-integrations.googleapis.com/v1/projects/%s/locations/%s/authConfigs/%s?updateMask=*&alt=json`

**Comment by ssvaidyanathan** [comment]
Thanks for checking @cesar-m-pinto-alb 
We will look into this and provide an update soon

**Comment by ssvaidyanathan** [comment]
We just released a beta version (v0.71.1-beta), can you please test using that and let us know if its working as expected

To install the beta version:
```sh
export INTEGRATIONCLI_VERSION=v0.71.1-beta
curl -L https://raw.githubusercontent.com/GoogleCloudPlatform/application-integration-management-toolkit/main/downloadLatest.sh | sh -
```

**Comment by ssvaidyanathan** [comment]
@cesar-m-pinto-alb - can you please try with `0.71.2-beta`

**Comment by cmgpbint** [comment]
it worked.
